packages/circuit-ui/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -91,17 +91,3 @@ Disabled.args = {
   value: 'true',
   disabled: true,
 };
-
-export const Multiple = (args: CheckboxProps) => (
-  <Fragment>
-    <CheckboxWithState {...args} value="apples" name="fruits">
-      Apples
-    </CheckboxWithState>
-    <CheckboxWithState {...args} value="bananas" name="fruits">
-      Bananas
-    </CheckboxWithState>
-    <CheckboxWithState {...args} value="oranges" name="fruits">
-      Oranges
-    </CheckboxWithState>
-  </Fragment>
-);
```

Comment:
Why did you remove this story? This is a common use case, so I believe it's important to document how to implement it properly (i.e. associating the checkboxes using the `name` attribute).

Reply:
🤦 I didn't think of the checkboxes-with-the-same-`name`-use-case.

Haven't seen it used a lot TBH (it's more common to have multiple checkboxes for multiple inputs, e.g. on auth.sumup.com for signup) but it's a valid point. Will add again